platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -224,6 +224,11 @@ public static native boolean initialize(Activity activity,
 	 */
 	public static native void onNightModeChanged();
 
+	/**
+	 * Invoked on the file picker closed.
+	 */
+	public static native void filePickerCallback(boolean p_ok, String[] p_selected_paths, int p_filter);
```

Comment:
@syntaxerror247 `0` is always passed for the `p_filter` argument, does it need to be included? Is there a scenario where we would pass different values?

Reply:
@m4gr3d Done!